src/components/ItemLookup.vue

```diff
@@ -15,18 +15,23 @@ interface Props {
 	placeholder: string;
 	value: SearchedItemOption | null;
 	searchForItems: ( searchTerm: string, offset?: number ) => Promise<SearchedItemOption[]>;
+	searchInput: string | null | undefined;
 	error?: { type: 'error'|'warning'; message: string } | null;
 	itemSuggestions?: SearchedItemOption[];
 }
 const props = withDefaults( defineProps<Props>(), {
 	error: null,
 	itemSuggestions: () => [],
+	searchInput: null,
 } );
 
 const emit = defineEmits( {
 	'update:modelValue': ( value: Props['value'] ) => {
 		return value === null || /^Q\d+$/.test( value.id );
 	},
+	'update:modelSearchInput': ( value: Props['searchInput'] ) => {
+		return true;
+	},
 } );
 
 const searchInput = ref( '' );
```

Comment:
I only noticed just now that we have two `searchInput`s here now: the prop and the ref. Pretty sure we should combine these in some way. (And I think we can actually remove the `watchEffect` below this line, and instead have the store control the `searchInput`s in `HANDLE_INIT_PARAMS`.)